Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -4,162 +4,142 @@
 using System;
 using System.Collections.Generic;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 using AzureCognitiveSearch.PowerSkills.Common;
-using Newtonsoft.Json.Linq;
 using System.Linq;
-using System.Collections;
-using System.IO;
 using System.Globalization;
-using System.Text;
-
-// languages used for Azure Search with Text Analytics:
-// el, th, he, tr, cs, hu, ar, ja-jp, fi, da, no, ko, pl, ru, sv, ja, it, pt, fr, es, nl, de, en
-// greek, thai, hebrew, turkish, czech, hungarian, arabic, japanese, finnish, danish, norwegian, korean, polish, russian, swedish, japanese (again??), 
-// italian, portuguese, french, spanish, dutch, german, english
-// unicode blocks in order:
-// InThai, InHebrew
+using System.Threading.Tasks;
 
 namespace AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch
 {
     /// <summary>
-    /// Based on sample custom skill provided in Azure Search. Provided a user-defined list of entities
-    /// this function determines the Entity first occurrence within a given document. This list of entities
-    /// must repeatedly be provided by the user for each document.
+    /// Based on sample custom skill provided in Azure Search. Given a user-defined list of entities,
+    /// this function will find all occurences of that entity in some input text.
     /// </summary>
     public static class CustomEntitySearch
     {
-        // Use this to load from "csv" or "json" file 
-        public static IList<string> preLoadedWords = null;
+        // ** Some global variables used for configuration.**
+        // ** Change these values prior to deploying your function **
+        public static bool ExactMatchesShouldBeCaseSensitive = false;
+        public static string EntityDefinitionLocation = "words.csv"; // other option is "words.json"
```

Comment:
Not new from this PR, but eventually, we should lookup an environment variable and let it override this default instead of requiring rebuilding the code.